### PR TITLE
[4755][IMP] connector_ebisumart: propagate ebisumart_send_date to invoice

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4755](https://www.quartile.co/web#model=project.task&id=4755&active_id=4755&cids=3)

### CODE REVIEW

File: connector_ebisumart/models/ebisumart_sale_order.py [hunk 1]
```diff
@@ -125,6 +125,14 @@ def after_import(self, ebisumart_record, backend_record):
         # TODO put this line after action_confirm()
         self.picking_ids.write({'scheduled_date': self.ebisumart_send_date})
 
+    @api.model
```
Comment (yostashiro) [inline connector_ebisumart/models/ebisumart_sale_order.py]:
```suggestion
    @api.multi
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — COMMENTED** [review]
As we just talked, please add the readme explaining the assumption on the OdooBot user.

**Comment by AungKoKoLin1997** [comment]
@kanda999 Please help to add the steps in Configuration for Ebisumart side. Eg, adding redirect url.

File: connector_ebisumart/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+This module connects Odoo with Ebisumart.
+Ebisumart (https://ebisumart.com/) is a popular Japanese e-commerce platform.
+
+This module allows the synchronization of the following objects from Ebisumart to Odoo:
+- Product
+- Partner
+- Sale Order
+- Purchase Order
```
Comment (yostashiro) [inline connector_ebisumart/readme/DESCRIPTION.rst:8]: It's misleading to include purchase orders (they don't come from Ebisumart; they are generated within Odoo).

@kanda999 Please confirm about partners as my memory is vague.

File: connector_ebisumart/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -2,6 +2,13 @@ To configure this module, you need to set several things in both PrestaShop and
 
 Steps in Ebisumart
 
+* Create the Ebisumart Apps.(https://support.ebisumart.com/category/APP/APP_INSTALL.html)
+    * Login to the Ebisumart Developer tool.
+    * Go to the Apps management page.
+    * Create a new Apps. Apps is required to connect with Odoo.
+
+* Need to log in to the ebisumart shop page, so please create an account for the shop page as well
```
Comment (AungKoKoLin1997) [inline connector_ebisumart/readme/CONFIGURE.rst:15]: @kanda999 I think we should mention about adding redirect url.

File: connector_ebisumart/models/ebisumart_sale_order.py [hunk 4]
```diff
@@ -117,6 +117,7 @@ def after_import(self, ebisumart_record, backend_record):
             'purchase_id': po.id,
         }
         inv = self.env['account.invoice'].create(po_invoice)
+        inv.write({'date_invoice': self.ebisumart_send_date})
```
Comment (kanda999) [inline connector_ebisumart/models/ebisumart_sale_order.py]: In this method, the previous day is recorded due to date and datetime fields. In the case of the UTC+9 time zone, even though ebisumart_send_date is a datetime field with October 18th 00:00:00, in the database it is recorded as October 17th. 
When this value is passed to a Date field, it gets recorded as October 17th. 
This issue does not occur with Customer Invoice

**Review by kanda999 — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]
Code review.

File: connector_ebisumart/README.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to set several things in both PrestaShop and Odoo:
+
+Steps in Ebisumart
+
```
Comment (kanda999) [inline connector_ebisumart/README.rst:44]: @AungKoKoLin1997 
Why is the readme not reflecting?

Comment (AungKoKoLin1997) [reply connector_ebisumart/README.rst:44]: Seem like pre-commit didn't generate readme. I will rerun.

Comment (AungKoKoLin1997) [reply connector_ebisumart/README.rst:44]: @kanda999 After this PR https://github.com/qrtl/sbt-custom/pull/11 is merged, I will rebase to regenerate README.rst.

Comment (kanda999) [reply connector_ebisumart/README.rst:44]: I merged it.

**Review by kanda999 — APPROVED** [review]
Code and Functional review.

**Pull request merged**